tools/java_api/src/main/java/com/kuzudb/QueryResult.java

```diff
@@ -18,11 +18,11 @@ private void checkNotDestroyed() throws ObjectRefDestroyedException {
     }
 
     /**
-     * Finalize.
+    * Close the queryResult and release the underlying resources. This method is invoked automatically on objects managed by the try-with-resources statement.
```

Comment:
Using `camelCase` does not make sense here. It should either be `query result` (plain English) or `QueryResult` (consistent with the class name)

Reply:
If you decide to change it to `QueryResult`, you need to change `database` -> `Database` etc. to be consistent in other comments

Reply:
So I'd suggest to just go with `query result`